packages/host/app/components/card-catalog/modal.gts

```diff
@@ -299,12 +323,18 @@ export default class CardCatalogModal extends Component<Signature> {
   debouncedSearchFieldUpdate = debounce(() => this.onSearchFieldUpdated(), 500);
 
   @action setCardURL(cardURL: string) {
-    this.selectedCard = undefined;
-    this.cardURL = cardURL;
+    if (!this.state) {
```

Comment:
why is it possible for the state to be undefined, isn't it created as part of construction?

Reply:
I missed to delete this. Updated [2281a83](https://github.com/cardstack/boxel/pull/616/commits/2281a83db15dd12a30e243f1082ca01539c0867a)